test/development/acceptance-app/ReactRefreshLogBox.test.ts

```diff
@@ -437,6 +488,20 @@ describe('ReactRefreshLogBox app', () => {
          "stack": [],
        }
       `)
+    } else if (isRspack) {
+      await session.assertHasRedbox()
```

Comment:
```suggestion
      // rspack returns error content that contains absolute paths which are non deterministic
      // using toContain instead of inline snapshot to check redbox content for correctness
      await session.assertHasRedbox()
```